apps/modernization-ui/src/pages/addPatient/AddPatient.tsx

```diff
@@ -100,8 +100,25 @@ const AddPatient = () => {
         setEntryState({ step: 'entry' });
     };
 
+    function isMissingFields(entry: NewPatientEntry) {
```

Comment:
The name is a little confusing because `isMissingFields` return `false` when when fields are missing and `true` when they all contain data.

Reply:
just replaced the boolean values as per the function name so it makes sense now